dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/NettyChain.java

```diff
@@ -131,155 +135,155 @@ public synchronized void stop() {
+                                .with(ConfigElement.SAMESITE,owner.getSamesiteConfig())
+                                .build();
 
                 bootstrap.childHandler(httpPipeline);
 
                 serverChannel = nettyFramework.start(bootstrap, info.getHost(), info.getPort(), this::channelFutureHandler);
 
+
                 VirtualHostMap.notifyStarted(owner, () -> currentConfig.getResolvedHost(), currentConfig.getConfigPort(), isHttps);
                 String topic = owner.getEventTopic() + HttpServiceConstants.ENDPOINT_STARTED;
                 postEvent(topic, currentConfig, null);
-                
-                
 
             } catch (Exception e) {
-                MSP.log("Failed to start NettyChannel: " + e.getMessage());
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.exit(this, tc, "Failed to start Netty Channel: " + e.getMessage());
+                }
                 state.set(ChainState.STOPPED);
-            } finally {
-                notifyAll();
             }
         }
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.exit(this, tc, "Finished starting Netty Channel: " + endpointName + ", Final state: " + state.get());
+        }
 
     }
 
     private void channelFutureHandler(ChannelFuture future) {
-        if (future.isSuccess()) {
-            state.set(ChainState.STARTED);
-            EndPointInfo info = endpointMgr.getEndPoint(this.endpointName);
-            info = endpointMgr.defineEndPoint(this.endpointName, currentConfig.configHost, currentConfig.configPort);
-            // VirtualHostMap.notifyStarted(owner, () -> currentConfig.getResolvedHost(), currentConfig.getConfigPort(), isHttps);
-            // String topic = owner.getEventTopic() + HttpServiceConstants.ENDPOINT_STARTED;
-            // postEvent(topic, currentConfig, null);
 
-            MSP.log("Channel is now active and listening on port " + getActivePort());
+        synchronized(this) {
+            if (future.isSuccess()) {
+                state.set(ChainState.STARTED);
+                EndPointInfo info = endpointMgr.getEndPoint(this.endpointName);
+                info = endpointMgr.defineEndPoint(this.endpointName, currentConfig.configHost, currentConfig.configPort);
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.debug(this, tc, "Channel is now active and listening on port " + getActivePort());
+                }
 
-        } else {
-            MSP.log("ChannelFutureHandler -> Failed to bind to port: " + future.cause());
-            stopAndWait();
+            } else {
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.debug(this, tc, "Channel failed to bind to port:  " + future.cause());
+                }
+                
+                stopAndWait();
```

Comment:
This should not be a stop wait. We should do the same logic as handleStartupError in HttpChain legacy